salt/journal/config/etc-nginx-sites-available-journal.conf

```diff
@@ -291,7 +291,46 @@ server {
     }
     {% endif -%}
 
-    {% if pillar.journal.about_url or pillar.journal.resources_url or pillar.journal.media_policy_url %}
+    {% if pillar.journal.community_url %}
+    # Proxy /community pages to pubpub
+    # - https://github.com/elifesciences/issues/issues/8516
+    # ali-tm-amin@2024-04:  Proxy all community pages below to pubpub
+    # "/community"                                 => proxy (as /community)
+    # "/community/"                                => proxy (as /community/)
+    # "/community/get-involved"                    => proxy (as /community/get-involved)
```

Comment:
each of these examples has the wrong proxy path for the right hand side

Reply:
Corrected, thanks